libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -165,9 +165,9 @@ class MapboxRouteLineView(var options: MapboxRouteLineOptions) {
      * @param style an instance of the Style
      * @param update an instance of VanishingRouteLineUpdateState
      */
-    fun renderVanishingRouteLineUpdateValue(
```

Comment:
The old method is still mentioned in the `MapboxRouteLineApi` class doc.

Reply:
I didn't catch that one. I want to remove those detailed docs at the top of the class because it's too hard to keep them up to date. They were created because we were making the SDK available to a customer and the standard docs weren't ready yet.  I will be updating the documentation pages with the latest updates to the route line API.